sysvabi64/sysvabi64.rst

```diff
@@ -0,0 +1,881 @@
+
+ABI
+   Application Binary Interface:
+
+   1. The specifications to which an executable must conform in order to
+      execute in a specific execution environment. For example, the
+      *Linux ABI for the Arm Architecture*.
+
+   2. A particular aspect of the specifications to which independently
+      produced relocatable files must conform in order to be
+      statically linkable and executable.  For example, the CPPABI64
+
+Arm-based
+   ... based on the Arm architecture ...
+
+Floating point
+   Depending on context floating point means or qualifies: (a) floating-point
+   arithmetic conforming to IEEE 754 2008; (b) the Armv8 floating point
+   instruction set; (c) the register set shared by (b) and the Armv8 SIMD
+   instruction set.
+
+Q-o-I
+   Quality of Implementation – a quality, behavior, functionality, or
+   mechanism not required by this standard, but which might be provided
+   by systems conforming to it.  Q-o-I is often used to describe the
+   tool-chain-specific means by which a standard requirement is met.
+
+SIMD
+   Single Instruction Multiple Data – A term denoting or qualifying:
+   (a) processing several data items in parallel under the control of one
+   instruction; (b) the Arm v8 SIMD instruction set: (c) the register set
+   shared by (b) and the Armv8 floating point instruction set.
+
+SIMD and floating point
+   The Arm architecture’s SIMD and Floating Point architecture comprising
+   the floating point instruction set, the SIMD instruction set and the
+   register set shared by them.
+
+SVE
+   The Arm architecture's Scalable Vector Extension.
+
+T32
+   The instruction set named Thumb in the Armv7 architecture; T32 uses
+   16-bit and 32-bit instructions.
+
+VG
+   The number of 64-bit “vector granules” in an SVE vector; in other words,
+   the number of bits in an SVE vector register divided by 64.
+
+ILP32
+   SysV-like data model where int, long int and pointer are 32-bit
+
+LP64
+   SysV-like data model where int is 32-bit, but long int and pointer are 64-bit.
+
+LLP64
+   Windows-like data model where int and long int are 32-bit, but long long int and pointer are 64-bit.
+
+This document uses the following terms and abbreviations.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+SysV
+   Unix System V. A variant of the Unix Operating System. Although
+   this specification refers to SysV, many other operating systems,
+   such as Linux or BSD use similar conventions.
+
+Platform
+   A program execution environment such as that defined by an
+   operating system or run- time environment. A platform defines the
+   specific variant of the ABI and may impose additional
+   constraints. Linux is a platform in this sense.
+
+More specific terminology is defined when it is first used.
+
+.. raw:: pdf
+
+   PageBreak
+
+Scope
+=====
+
+Except where otherwise stated the AArch64 System V ABI follows the
+AArch64 base ABI documents in ABIAA_ . The AArch64 System V ABI
```

Comment:
We don't refer to the abi documentation as `ABIAA`, yet this is how this gets rendered in the documentation.

Reply:
I've changed to use "ABI for the Arm 64-bit Architecture" with a link to the main page.